build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+set -ueo
+
+MSQUIC_VERSION="$1"
+TARGET_SO='priv/libquicer_nif.so'
+PKGNAME="$(./pkgname.sh)"
+
+build() {
+    # default: 4 concurrent jobs
+    JOBS=4
+    if [ "$(uname -s)" = 'Darwin' ]; then
+        JOBS="$(sysctl -n hw.ncpu)"
+    else
+        JOBS="$(nproc)"
+    fi
+    ./get-msquic.sh "$MSQUIC_VERSION"
+    cmake -B c_build
+    make -j "$JOBS" -C c_build
+    ## MacOS
+    if [ -f priv/libquicer_nif.dylib ]; then
+        # TODO: why cp, but not mv ?
+        cp priv/libquicer_nif.dylib "$TARGET_SO"
+    fi
+}
+
+download() {
```

Comment:
I prefer not to have download as default since there are lots of customization in the emqx build image that the binary built with them might not work with upstream images.

If you really want download to be default, pls `echo` loudly so developers know it is downloading from private build also have envvar for developer to have option  to compile instead.